app/controllers/application_controller.rb

```diff
@@ -35,6 +35,7 @@ def current_stimulus_triage
   end
 
   def clear_intake_session
+    sign_out current_client
     session.delete("intake_id")
```

Comment:
I thought that it made sense to automatically sign out the client at the point that we would normally clear their session.

Reply:
I think this implementation fits the scope of this ticket! Do you know if there is a follow up ticket for what to do with authenticated clients once they've finished intake? If not, it might be worth putting on product's radar!

Reply:
There is not currently any follow up ticket, so yeah, that's a great suggestion! I'll bring it up with them.